File: src/components/pages/projects/$projectId/shared/connect-snippets/deno/fresh/lib/appwrite.ts
import { Client } from "{{DENO_SDK_SPECIFIER}}"

const client = new Client()
  .setEndpoint(Deno.env.get("APPWRITE_ENDPOINT")!)
  .setProject(Deno.env.get("APPWRITE_PROJECT_ID")!)
  .setKey(Deno.env.get("APPWRITE_API_KEY")!)

export { client }

File: src/components/pages/projects/$projectId/shared/connect-snippets/dart/serverpod/lib/src/appwrite.dart
import 'dart:io' show Platform;

import 'package:dart_appwrite/dart_appwrite.dart';

final appwriteClient = Client()
    .setEndpoint(Platform.environment['APPWRITE_ENDPOINT']!)
    .setProject(Platform.environment['APPWRITE_PROJECT_ID']!)
    .setKey(Platform.environment['APPWRITE_API_KEY']!);

final appwriteProject = Project(appwriteClient);
